nginx/modsecurity/ngx_http_modsecurity.c

```diff
@@ -739,6 +739,16 @@ ngx_http_modsecurity_detection_thread_completion(ngx_event_t *ev)
     ngx_http_request_t *r = ev->data;
     /* 'blocked' is incremented in ngx_http_modsecurity_detection_task_offload */
     --r->main->blocked;
+    /* This is to handle the case when the request is prematurely closed by the client.
```

Comment:
Minor nit:
s/request is prematurely/connection is prematurely

Reply:
Addressed.